charts/argo/templates/_helpers.tpl

```diff
@@ -14,3 +14,23 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{- define "workflow-crd-json" }}
```

Comment:
I personally don't like the workflow-crd.json to be placed in the helpers.tpl because it's no real helper but an important resource definition.
Maybe we could place it it in an separate templates/etc folder as _workflow-crd.json and create the resource from there?

Reply:
I agreed it's better to have it separated out, I'll make it it's own `tpl` file

Reply:
done